tests/unit/converters/test_vision_excel_converter.py

```diff
@@ -32,22 +32,59 @@ def converter() -> VisionExcelConverter:
     return converter
 
 
-@pytest.mark.parametrize("language", ["en"])
+@pytest.fixture
+def converter_nl() -> VisionExcelConverter:
+    # Arrange
+    converter = VisionExcelConverter(language="nl")
+    converter._get_id("Knooppunten", {"Nummer": 1}, None)  # node: 0
+    converter._get_id("Kabels", {"Nummer": 1}, None)  # branch: 1
+    converter._get_id("Links", {"Nummer": 1}, None)  # branch: 2
+    converter._get_id("Smoorspoelen", {"Nummer": 1}, None)  # branch: 3
+    converter._get_id("Speciale transformatoren", {"Nummer": 1}, None)  # branch: 4
+    converter._get_id("Transformatorbelastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, "transformer")  # virtual: 5
+    converter._get_id("Transformatorbelastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, "internal_node")  # virtual:  6
+    converter._get_id("Transformatorbelastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, "load")  # virtual:  7
+    converter._get_id("Transformatorbelastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, "generation")  # virtual:  8
+    converter._get_id("Transformatorbelastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, "pv_generation")  # virtual:  9
+    converter._get_id("Netvoedingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 10
+    converter._get_id("Synchrone generatoren", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 11
+    converter._get_id("Windturbines", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 12
+    converter._get_id("Belastingen", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 13
+    converter._get_id("Nulpuntstransformatoren", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 14
+    converter._get_id("Pv's", {"Knooppunt.Nummer": 1, "Subnummer": 2}, None)  # appliance: 15
+    return converter
+
+
+@pytest.mark.parametrize("language", ["en", "nl"])
 def test_mapping_files_exist(language: str):
     vf = Path(str(DEFAULT_MAPPING_FILE).format(language=language))
     assert vf.exists()
 
 
-def test_initialization():
+def test_mapping_file_none_input_raises_file_not_found_error():
```

Comment:
`language` should be a string, so there we don't have to test this. It's up to the user to supply to respect the type hints.